pkg/util/funcmap/extra_funcs.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2021 The Operating System Manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package funcmap
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"text/template"
+
+	"github.com/BurntSushi/toml"
+	"github.com/Masterminds/sprig/v3"
+	"gopkg.in/yaml.v2"
```

Comment:
Sorry for getting back to you so late, but why is this using `yaml.v2`? We are already using `yaml.v3`, as seen in the `go.mod` file.

Reply:
Used `v3` instead `v2` in latest commit.